### PR TITLE
cli: disable old certs flags

### PR DESCRIPTION
**Pull request by mberhault**
This can still be re-enabled by setting
COCKROACH_ALLOW_OLD_CERTS_FLAGS=true but the flags are now hidden.

This leaves people one last chance to remove them, having previously
received a warning about deprecation.

### CODE REVIEW

File: pkg/cli/interactive_tests/test_secure.tcl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 source [file join [file dirname $argv0] common.tcl]
 
-set certs_dir "certs"
+set certs_dir "/certs"
```
Comment (mberhault) [inline pkg/cli/interactive_tests/test_secure.tcl]: this did work, but we're using the absolute path in other places, I had it switched when testing manually a few PRs ago, this is just resetting it.

**Comment by mberhault** [comment]
@bdarnell: I believe this is our plan, it's even mentioned in the 0420 release notes. I'm happy to hold off on this if we want to have a grace period of an extra release.

**Review by a-robinson — APPROVED** [review]
It'd make sense for this to be in the first release candidate, which I believe is going to be next week, so LGTM.

**Review by bdarnell — APPROVED** [review]
LGTM, but I would go ahead and get rid of the env var and hidden flags too. If anyone gets caught by this they can just continue to run 0420 until they have adjusted their cert locations.

**Comment by mberhault** [comment]
ok, I've dropped all mention of the old flags in the second commit. Are we sure we want this?
Are we basically saying "beta versions support the flags, release candidates do not"?

**Comment by bdarnell** [comment]
LGTM. We are in effect saying that just because this week happens to be the first release candidate, but that's just coincidental. 

If we want a smoother transition, we could leave the flags in place for a longer period of time (and as I've said before I'd be fine with leaving them in forever). A second stage of transition in which the flags are usable only with an environment variable just seems kind of silly - why would someone add this environment variable instead of just updating to the new flags?

**Comment by mberhault** [comment]
Good point, if they're setting an env variable, setting the flag (or even just symlinking) is about the same.
Merging.

**Pull request merged**